src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -63,6 +65,31 @@ using namespace Yuni;
 #define SNPRINTF snprintf
 #endif
 
+class TimeMeasurement {
+    using clock = std::chrono::steady_clock;
+public:
+    TimeMeasurement() {
+        start_ = clock::now();
+        end_ = start_;
+    }
+    
+    void tick() {
+        end_ = clock::now();
+    }
+
+    long long duration_ms() const {
+        return std::chrono::duration_cast<std::chrono::milliseconds>(end_ - start_).count();
+    }
+
+    std::string toString() const {
+        return std::to_string(duration_ms()) + "ms";
+    }
```

Comment:
Add space between number and units, see rule 15 [here](https://physics.nist.gov/cuu/Units/checklist.html).